### ISSUE TITLE
Validation of Grantee

### ISSUE THREAD

**Issue by davidkurtz** [accasey/PSDbPrivs#7]
You can enter a grant to any user or role whether it exists or not.  I think that the grantee should be validated against DBA_USERS and DBA_ROLES.  
To be able to create a database view on these two catalogue views will require select privilege to be granted on them to SYSADM (via the PSADMIN role).
It is good practice to grant privileges to a role and then the role to users - the exception being privileges required by PL/SQL objects must be granted directly.

**Comment by accasey** [comment]
I agree, it wasn't done at first just due to a dependency on the Oracle DBA views .
